config.example.yaml

```
@@ -0,0 +1,137 @@
+# Mnemonic phrase, only one of this or key should be set
+mnemonic: "12 or 24 mnemonic words"
+
+# Number of excess wallets for submitting txs, required only when mnemonic option is used
+walletCount: 5
+
+# Topup amount for excess accounts, required only when mnemonic option is used
+topupAmount: 0.5
+
+# List of rpc urls, required
+# for specifying more than 1 RPC in the env, separate them by a comma and a space
+rpc: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)